storage/storage/src/main/java/one/microstream/storage/types/StorageConfiguration.java

```diff
@@ -194,7 +194,7 @@ public String toString()
 				.add(this.housekeepingController).lf()
 				.add(this.entityCacheEvaluator  ).lf()
 				.add(this.dataFileEvaluator     ).lf()
-				.add(this.backupSetup           ).lf()
+				.add(this.backupSetup == null ? "one.microstream.storage.types.StorageBackupSetup: null " : this.backupSetup).lf()
```

Comment:
`StorageBackupSetup.class.getName() + ": null"` would be better than `"one.microstream.storage.types.StorageBackupSetup: null "`

Reply:
True, ok